projects/plugins/beta/src/admin/plugin-manage.template.php

```diff
@@ -194,9 +194,10 @@ class="is-primary jp-form-button dops-button is-primary is-compact jptracks"
 		</div>
 		<div id="section-pr">
 			<?php
-			end( $manifest->pr );
-			$last = key( $manifest->pr );
-			foreach ( $manifest->pr as $k => $pr ) {
+			$pr_list = (array) $manifest->pr;
+			end( $pr_list );
+			$last = key( $pr_list );
+			foreach ( $pr_list as $k => $pr ) {
```

Comment:
If we don't care about being able to run with WordPress < 5.9 on PHP < 7.3, we could just do
```suggestion
			$last = array_key_last( (array) $manifest->pr );
			foreach ( $manifest->pr as $k => $pr ) {
```

Reply:
Since the beta plugin is publicly available, I think it needs to remain PHP 5.6+ compatible.

Reply:
https://github.com/WordPress/WordPress/blob/master/wp-includes/compat.php#L416

Reply:
Ah, I didn't know about the polyfill! 👍